Add New Featured Artist: AAAA news post (#7991)
ppy/osu-wiki: news/2022-09-11-new-featured-artist-aaaa.md

---
layout: post
title: "New Featured Artist: AAAA"
date: 2022-09-11 16:30:00 +0000
---

WELCOME **AAAA** TO OUR FEATURED ARTIST LIBRARY.

![](https://assets.ppy.sh/artists/301/header.jpg)

> Earning the spotlight in games like *Arcaea*, *maimai*, *CHUNITHM*, *pop'n music*, SDVX*, *Muse Dash*, and... more than we can reasonably list, **AAAA** is an undeniable legend in the realm of rhythm game compositions.

Reviewer: ```suggestion
Earning the spotlight in games like *Arcaea*, *maimai*, *CHUNITHM*, *pop'n music*, *SDVX*, *Muse Dash*, and... more than we can reasonably list, **AAAA** is an undeniable legend in the realm of rhythm game compositions.
```

**22** insanely awesome tracks are ready for mapping from [**AAAA**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/301)!

> **AAAA**'s Featured Artist reveal was a but unorthodox. In collaboration with fellow legends [Hino Isuka](https://osu.ppy.sh/beatmaps/artists/272) and [Umebsohi Chazuke](https://osu.ppy.sh/beatmaps/artists/98), **AAAA** helped create the custom song [*steganography*](LINK) for the MWC 4K 2022 Tiebreaker!

Reviewer: ```suggestion
**AAAA**'s Featured Artist reveal was a bit unorthodox. In collaboration with fellow legends [Hino Isuka](https://osu.ppy.sh/beatmaps/artists/272) and [Umeboshi Chazuke](https://osu.ppy.sh/beatmaps/artists/98), **AAAA** helped create the custom song [*steganography*](LINK) for the MWC 4K 2022 tiebreaker!
```

If you missed it live, watch how the reveal went in the video below, then check out the massive collection of beatmaps featuring **AAAA** tunes:

<div align="center">
    <video width="100%" controls>
        <source src="https://assets.ppy.sh/artists/301/release_showcase.mp4" type="video/mp4" preload="none">
    </video>
</div>

### aa...hisuichazuke... - steganography

> Play through [the MWC 4K 2022 Tiebreaker](https://osu.ppy.sh/beatmapsets/1846639#mania/3793375) for yourself, mapped by [-mint-](https://osu.ppy.sh/users/8976576), [guden](https://osu.ppy.sh/users/11626065), [MyZterioN-](https://osu.ppy.sh/users/8521723), and [Toaph Daddy](https://osu.ppy.sh/users/7616811)!

Reviewer: ```suggestion
Play through [the MWC 4K 2022 tiebreaker](https://osu.ppy.sh/beatmapsets/1846639#mania/3793375) for yourself, mapped by [-mint-](https://osu.ppy.sh/users/8976576), [guden](https://osu.ppy.sh/users/11626065), [MyZterioN-](https://osu.ppy.sh/users/8521723), and [Toaph Daddy](https://osu.ppy.sh/users/7616811)!
```

---
layout: post
title: "New Featured Artist: AAAA"
date: 2022-09-11 16:30:00 +0000
---

WELCOME **AAAA** TO OUR FEATURED ARTIST LIBRARY.

![](https://assets.ppy.sh/artists/301/header.jpg)

Earning the spotlight in games like *Arcaea*, *maimai*, *CHUNITHM*, *pop'n music*, *SDVX*, *Muse Dash*, and... more than we can reasonably list, **AAAA** is an undeniable legend in the realm of rhythm game compositions.

**22** insanely awesome tracks are ready for mapping from [**AAAA**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/301)!

> **AAAA**'s Featured Artist reveal was a bit unorthodox. In collaboration with fellow legends [Hino Isuka](https://osu.ppy.sh/beatmaps/artists/272) and [Umeboshi Chazuke](https://osu.ppy.sh/beatmaps/artists/98), **AAAA** helped create the custom song [*steganography*](LINK) for the MWC 4K 2022 tiebreaker!

Reviewer 2: It's missing a link here (to the song file/listing?)

Reviewer 3: unsure if markdown supports greek letters but the e in steganography is actually epsilon "ε"

Reviewer: iirc news posts use the romanised title
